commands/init/deployer/deployer.ts

```diff
@@ -50,45 +38,34 @@ export class Deployer {
     }
 
     async deployContract(contractWasm: string) {
-        const contractDataBase64 = fs.readFileSync(contractWasm, "base64");
+        const binaryArrayBuffer: Buffer = fs.readFileSync(contractWasm, {});
+        const binaryFileContents = new Uint8Array(binaryArrayBuffer);
+        const base64: string = Buffer.from(binaryFileContents).toString("base64");
```

Comment:
Do we still deploy SC encoded in base64 ?

Reply:
yes, otherwise we get this error: 

`TypeError: Cannot read properties of undefined (reading 'length')
    at SmartContractsClient.<anonymous> (/Users/nathanmassa/dev/massa-sc-toolkit/commands/init/node_modules/@massalabs/massa-web3/src/web3/SmartContractsClient.ts:53:39)
`

Reply:
@AurelienFT do you know why?

Reply:
I think it's more on @Eitu33 side.

Reply:
See with @SebastienValla which had the same problem with the transformer if I remember well and maybe @Eitu33 yes

Reply:
I don't understand why I am getting pinged about a web3 module implementation error, can I have more context?

Reply:
because you are THE MAN for the job. I close this issue because we need to move forward, but we will need your help and thus concat  you via discord.

Reply:
Ah ok I see, FYI the base64 parameter comes from Evegni's `ISmartContract` interface type (from memory). It has nothing to do with the core update about using bytes arrays, if you wish to have a more explicit parameter you should change the `deploySmartContract` functions and the SC interface object I believe